tests/kafkatest/tests/client/client_compatibility_features_test.py

```diff
@@ -130,4 +134,7 @@ def run_compatibility_test(self, broker_version, metadata_quorum=quorum.zk):
         self.kafka.set_version(KafkaVersion(broker_version))
         self.kafka.start()
         features = get_broker_features(broker_version)
+        if not self.zk:
+            # this is supported by zkBroker currently
+            features["describe-acls-supported"] = False
```

Comment:
Nice -- better than disabling the test for Raft-based quorums.  Perhaps add to the comment to indicate that this check/disabling is only necessary due to the fact that we are in early access mode with KIP-500 and we should remove the special casing when that his fully implemented?  Aside from that, LGTM -- thank you for the PR!

Reply:
will copy that.